### PR TITLE
Feature/asv 262 app view card

### PR DESCRIPTION
**Pull request by analaragomes**
**What does this PR do?**

   All preview app cards should be same. 
Unfortunately, it was not possible to use the same layout everywhere.
There are two layouts: with ads and witout ads.
Where are the cards?
- home (ads and other bundles)
- home + more of trending bundle
- home + more of ads bundle
- home + more of if snipe it bundle
- app view, press install => Similar app bundles has the two types of cards:

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppViewSuggestedAppDisplayable.java
- [ ] AppViewSuggestedAdDisplayable.java
- [ ] GridAppDisplayable.java
- [ ] GridAdDisplayable.java.java
- [ ] AppViewSuggestedAppWidget.java
- [ ] AppViewSuggestedAdWidget.java
- [ ] GridAppWidget.java
- [ ] GridAdWidget.java

**How should this be manually tested?**

Where are the cards?
- home (ads and other bundles)
- home + more of trending bundle
- home + more of ads bundle
- home + more of if snipe it bundle
- app view, press install => Similar app bundles has the two types of cards: with and without adds

Known problem:
- [ ] AppViewSuggestedAppDisplayable.java
- [ ] GridAppDisplayable.java
and
- [ ] AppViewSuggestedAdDisplayable.java
- [ ] GridAdDisplayable.java.java
we should be able to re-use the layout.xml, but we can't

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-296
https://aptoide.atlassian.net/browse/ASV-262

**Questions:**

   Does this add new dependencies which need to be added? No

**Code Review Checklist**

- [ ] Architecture
- [ ] Documentation on public interfaces
- [ ] Database changed?
- [ ] If yes - Database migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] Partners build
- [ ] Unit tests pass
- [ ] Functional QA tests pass

### CODE REVIEW

File: app/src/main/java/cm/aptoide/pt/app/view/GridAppDisplayable.java [hunk 1]
```diff
@@ -32,7 +31,6 @@ public GridAppDisplayable(App pojo, String tag, boolean totalDownloads,
       NavigationTracker navigationTracker, StoreContext storeContext) {
     super(pojo);
     this.tag = tag;
-    this.totalDownloads = totalDownloads;
```
Comment (JDSM01) [inline app/src/main/java/cm/aptoide/pt/app/view/GridAppDisplayable.java]: You forgot to remove it from the constructor

File: app/src/main/java/cm/aptoide/pt/app/view/GridAppWidget.java [hunk 2]
```diff
@@ -51,21 +46,18 @@ public GridAppWidget(View itemView) {
...
-        .getTimeDiffString(context, pojo.getAdded()
-            .getTime(), getContext().getResources()));
+
+    try {
+      DecimalFormat oneDecimalFormatter = new DecimalFormat("#.#");
+      rating.setText(oneDecimalFormatter.format(pojo.getStats()
+          .getRating()
+          .getAvg()));
+    } catch (Exception e) {
+      rating.setText(R.string.appcardview_title_no_starts);
```
Comment (JDSM01) [inline app/src/main/java/cm/aptoide/pt/app/view/GridAppWidget.java]: Shouldn't the string be called ...no_stars?

File: app/src/main/java/cm/aptoide/pt/app/view/GridAppDisplayable.java (hunk 1 above)
Comment (analaragomes) [reply app/src/main/java/cm/aptoide/pt/app/view/GridAppDisplayable.java]: done

File: app/src/main/java/cm/aptoide/pt/app/view/GridAppWidget.java (hunk 2 above)
Comment (analaragomes) [reply app/src/main/java/cm/aptoide/pt/app/view/GridAppWidget.java]: done

**Pull request merged**